src/_gettsim/transfers/wohngeld.py

```diff
@@ -3,6 +3,14 @@
 from _gettsim.config import numpy_or_jax as np
 from _gettsim.piecewise_functions import piecewise_polynomial
 from _gettsim.shared import policy_info
+from _gettsim.transfers.wohngeld_formulas import (
```

Comment:
I am not a fan of this structure. I know this module will grow large, but there is a lot of value in seeing the formula right next to the code. Having these internal functions instead of copying and pasting the code is good, though.